README.md

```diff
@@ -1,7 +1,7 @@
 # Scalding
 
 [![Build status](https://github.com/twitter/scalding/actions/workflows/CI.yml/badge.svg?branch=develop)](https://github.com/twitter/scalding/actions)
-[![Coverage Status](https://coveralls.io/repos/twitter/scalding/badge.png?branch=develop)](https://coveralls.io/r/twitter/scalding?branch=develop)
+[![Coverage Status](https://img.shields.io/codecov/c/github/twitter/scalding/develop.svg?maxAge=3600)](https://codecov.io/github/twitter/scalding)
```

Comment:
max age is 1h? seems like maybe that should be longer no?

Reply:
I'm fine with whatever. Although github also does their own caching so it's probably a bit irrelevant